tensorhive/app/web/AppServer.py

```diff
@@ -50,11 +50,13 @@ def _inject_api_endpoint_to_app():
     try:
         web_app_json_config_path = PosixPath(__file__).parent / 'dist/static/config.json'
         data = {
-            'apiPath': 'http://{}:{}/{}'.format(
+            'apiPath': '{}://{}:{}/{}'.format(
+                API.URL_SCHEMA,
                 API.URL_HOSTNAME,
-                API_SERVER.PORT,
+                API.URL_PORT,
                 API.URL_PREFIX),
-            'version': tensorhive.__version__
+            'version': tensorhive.__version__,
+            'baseURL': APP_SERVER.BASE_URL
```

Comment:
It seems to me that baseURL is no longer needed.
@tboinski can you confirm that?

Reply:
I think it is, as it injects proper paths to config.json. I had some problems when I didn't change the baseUrl value.

Reply:
In the current version baseUrl is not used, as it has been deleted in a1c6f7413ef7709d18918caca2046e36ef60dbe2
Can you remember what kind of problems you had? I will check the current version on a non-proxy deployment.

Reply:
If I didn't set it in the main-config the config.json was not properly injected and thus the server did not listen on the proper address.

Reply:
In the current version, the injected value is not used on the client side, so my suggestion is to remove it from both config.json and main_config.json. The question is if I should do it or if should I re-introduce it into main.js. In a non-proxy deployment, the current version works perfectly fine.

Reply:
Ok, I've checked and it looks that with hash router the variable is not necessary.

Reply:
OK, confirmed on two deployments both with and without reverse proxy. I will remove baseUrl soon.